knexfile.js

```diff
@@ -26,19 +56,18 @@ module.exports = {
   },
 
   production: {
-    client: 'postgresql',
-    connection: {
-      database: 'my_db',
-      user:     'username',
-      password: 'password'
-    },
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
```

Comment:
Make sure you update the README with info on any environment variables

Reply:
We have updated the README with this info!